static/js/components/ThumbnailList.jsx

```diff
@@ -111,6 +111,10 @@ const Thumbnail = ({ ra, dec, name, url, size, grayscale }) => {
             className={imgClasses}
             title={alt}
             loading="lazy"
+            onError={(e) => {
+              e.target.onerror = null;
+              e.target.src = "/static/images/outside_survey.png";
+            }}
```

Comment:
```suggestion
            onError={(e) => {
              if (url !== "#") {
                e.target.onerror = null;
                e.target.src = "/static/images/outside_survey.png";
              }
            }}
```

Reply:
this will prevent showing the "outside survey" image while the PS1 thumbnail is being generated